model.py

```diff
@@ -600,6 +600,7 @@ class DataSource(Base):
     GUTENBERG_COVER_GENERATOR = "Gutenberg Illustrated"
     GUTENBERG_EPUB_GENERATOR = "Project Gutenberg EPUB Generator"
     METADATA_WRANGLER = "Library Simplified metadata wrangler"
+    METADATA_WRANGLER_COLLECTION = "Library Simplified Metadata Wrangler Collection"
```

Comment:
See my comment on https://github.com/NYPL-Simplified/circulation/pull/167/files about this data source name.